### DISCUSSION TITLE
M2A relation counterpart(O2M)?

### DISCUSSION THREAD

**Discussion by JonL1** [directus/directus#10504] [Archive]
When you create a M2O relation a counterpart(O2M) can be created from the UI, but I think this will not work for a M2A as the UI will not allow to query using the  'item' and 'collection' columns from the junction table, right?

**Comment by rijkvanzanten** [comment]
That's correct. Directus doesn't currently support the "opposite" relationship of an M2A. 

See https://github.com/directus/directus/discussions/9350
